app/store/index.js

```diff
@@ -124,20 +124,21 @@ export default new Vuex.Store({
 			else if (state.userType === "Food Bank") {
 				idType = "foodBankId"
 			}
-
-			bindFirestoreRef('activeOrders', db.collection("Orders").where(idType, "==", state.id)
-				.where('status', 'in', ['Looking For Driver', 'Driver on route for pick up', 'Inventory picked up']))
+			bindFirestoreRef('activeOrders', db.firestore().collection("Orders").where(idType, "==", state.id))
```

Comment:
we need the additional .where('status', 'in', ['Looking For Driver', 'Driver on route for pick up', 'Inventory picked up']) also